src/colvar_gpu_calc.cpp

```diff
@@ -446,9 +446,7 @@ int colvarmodule_gpu_calc::calc_cvs(const std::vector<colvar*>& colvars, colvarm
   // Update flags
   checkColvarsError(cv_update_flags(colvars));
   // Calculate total force
-  if (colvar_module->step_relative() > 0) {
-    checkColvarsError(cvc_calc_total_force(colvars, colvar_module, false));
-  }
+  checkColvarsError(cvc_calc_total_force(colvars, colvar_module, false));
```

Comment:
Agreed with removing the additional if, since the function checks internally for the "valid" flag, but maybe add a 1-line comment here to mention that?